db/versions/0009.yml

```diff
@@ -189,7 +189,7 @@ methods:
             encode_string_key(cache_purges.cache_name) as row_key,
             jsonb_build_object(
               'PartitionKey', encode_composite_key(cache_purges.provisioner_id, cache_purges.worker_type),
-              'RowKey', cache_name,
+              'RowKey', encode_string_key(cache_name),
```

Comment:
Thanks :)

Reply:
Note that typically we can't fix old migrations, as they've already run and won't run again.  It looks like we got away with it in this case (I have a sneaking suspicion that the PartitionKey and RowKey JSON properties are never used?), and fixing this will help anyone referring to this version to not duplicate the bug.